apps/mosaico/src/Article/Article.purs

```diff
@@ -275,6 +284,68 @@ render props =
                 }
               ]
           }
+        vnPremium =
```

Comment:
These could actually live somewhere near `Vetrina`

Reply:
I added a new module `Vetrina.Products.Premium` to hold these. Ideally we'd get these from the server.